File: java-sample/src/main/java/io/snabble/testapp/ShoppingCartActivity.java
package io.snabble.testapp;

import androidx.fragment.app.Fragment;

import io.snabble.sdk.ui.cart.ShoppingCartFragment;

public class ShoppingCartActivity extends BaseActivity  {
    @Override
    public Fragment onCreateFragment() {
        return new ShoppingCartFragment();
    }
}

File: ui/src/main/java/io/snabble/sdk/ui/cart/deprecated/ShoppingCartActivity.kt
package io.snabble.sdk.ui.cart.deprecated

import androidx.fragment.app.Fragment
import io.snabble.sdk.ui.BaseFragmentActivity

@Deprecated("Integrate [io.snabble.sdk.ui.cart.shoppingcart.ShoppingCartScreen] instead.")
class ShoppingCartActivity : BaseFragmentActivity() {
    override fun onCreateFragment(): Fragment = ShoppingCartFragment()
}
